test/integration/full-stack/serverspec/nginx_spec.rb

```diff
@@ -7,3 +7,11 @@
   end
 
 end
+
+describe file('/etc/nginx/sites-available/intercity_sample_app.conf') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.